features/signal/child_workflow/feature.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from temporalio import workflow
+from temporalio.client import WorkflowHandle
+
+from harness.python.feature import Runner, register_feature
+
+CHILD_WORKFLOW_INPUT = "child-input"
+CHILD_WORKFLOW_INITIAL_STATE = "1"
+SIGNAL_ARG = "2"
+
+
+@workflow.defn
+class Workflow:
+    @workflow.run
+    async def run(self) -> str:
+        child_wf = await workflow.start_child_workflow(
+            ChildWorkflow.run, CHILD_WORKFLOW_INPUT
+        )
+        await child_wf.signal("my_signal", SIGNAL_ARG)
```

Comment:
```suggestion
        await child_wf.signal(ChildWorkflow.my_signal, SIGNAL_ARG)
```

I think the type safe form is better here